synapse/pipeline.yml

```diff
@@ -325,6 +325,56 @@ steps:
         - exit_status: 2
           limit: 2
 
+  - label: "synapse_port_db / :python: 3.5 / :postgres: 9.5"
+    agents:
+      queue: "medium"
+    env:
+      TRIAL_FLAGS: "-j 8"
```

Comment:
unless the test uses `trial` (I don't think it does?) there is no point in this, and no point in the artifact upload pattern below.